sql/hive/src/test/scala/org/apache/spark/sql/HiveSQLConfSuite.scala

```diff
@@ -0,0 +1,28 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.spark.sql
+
+import org.apache.spark.sql.hive.test.TestHiveSingleton
+
+class HiveSQLConfSuite extends TestHiveSingleton {
+
+  test("SPARK-36421: Validate all SQL configs to prevent from wrong use for ConfigEntry") {
```

Comment:
this probably doesn't worth a new suite. how about just test it in `org.apache.spark.sql.hive.execution.SQLQuerySuite`?

Reply:
updated